src/utils/helpers/index.ts

```diff
@@ -4,9 +4,14 @@ import { Pathnames } from 'libs/routing';
 import { twMerge } from 'tailwind-merge';
 import Graphemer from 'graphemer';
 import { TradePair } from 'libs/modals/modals/ModalTradeTokenList';
+import config from 'config';
 export * from './number';
 
 export const isProduction = window
+  ? window.location.host.includes(config.productionHostName)
+  : true;
+
+export const isNativeApp = window
```

Comment:
I'm not sure what's the best way to handle that, should we see the powerBy on development ?
Maybe having a debug flag might be useful to easily see how it looks like. To be discussed